src/Check.hs

```diff
@@ -90,13 +91,30 @@ checkTestsBuildReport False =
 checkTestsBuildReport True =
   "- The tests defined in `passthru.tests`, if any, passed"
 
+versionWithoutPath :: String -> Text -> String
+versionWithoutPath resultPath expectedVersion = do
+  -- We want to match expectedVersion, except when it is preceeded by
+  -- the new store path (as wrappers contain the full store path which
+  -- often includes the version)
+  -- This can be done with negative lookahead e.g
+  -- /^(?!.*${storePathWithoutVersion}).*${version}/
+  -- Note we also replace '.' with '\\.' for literal match in grep -P,
+  -- there might be a better way of escaping everything...
+  let storePath = maybe (T.pack resultPath) id $ T.stripPrefix (T.pack "/nix/store/") (T.pack resultPath)
```

Comment:
`T.pack` on string literals is useless here because the language extension `OverloadedStrings` is enabled on this file (see the top of the file).